changelog/july2023/2023-07-06-secret-manager-added-secret-manager-metrics-in.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Secret Manager Metrics in Cockpit
+status: added
+author:
+    fullname: 'Join the #secret-manager channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-07-06
+category: security-identity
+product: secret-manager
+---
+
+Secret Manager metrics are now displayed in Cockpit by default, allowing you to have more control over your metrics. Read our documentation to find out which other products are integrated into Cockpit.
```

Comment:
add a link to doc? 
```suggestion
Secret Manager metrics are now displayed in Cockpit by default, allowing you to have more control over your metrics. Read our [documentation](https://www.scaleway.com/en/docs/observability/cockpit/reference-content/cockpit-limitations/) to find out which other products are integrated into Cockpit.
```

Not sure if this is the right link @nerda-codes

Reply:
Yes it's the right link, but the doc is not up-to-date, I'm waiting for the list of the new product integrations to update it